tests/contrib/hooks/test_aws_glue_catalog_hook.py

```diff
@@ -20,17 +20,13 @@
 
 import boto3
 from airflow.contrib.hooks.aws_glue_catalog_hook import AwsGlueCatalogHook
+from tests.compat import mock
 
 try:
     from moto import mock_glue
 except ImportError:
     mock_glue = None
```

Comment:
We shouldn't need these anymore either should we - moto is in the requirements isn't it?

Reply:
(Not that we have to change that here if it's not causing any type issues. Lets not do it here actually)